source/_docs/relaunch.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Relaunch Existing Pantheon Site
+description: Take a new site live by moving custom domains from one Site Dashboard to another, with minimal HTTPS interruptions.
+tags: [dns]
+categories: []
+---
+Sites are considered launched on Pantheon once traffic is routed through custom domain(s). Relaunching a previously launched site is done by rerouting traffic from the existing Site Dashboard to an entirely new Site Dashboard.
+
+<div class="alert alert-info">
+<h4 class="info">Note</h4>
+<p markdown="1">The relaunch process applies exclusively to live sites already hosted on Pantheon. Otherwise, refer to [Launch Essentials](/docs/guides/launch/).</p></div>
+
+## Before You Begin
+- Log in to the new Site Dashboard on Pantheon
+- Open a second tab for the old Site Dashboard on Pantheon
+- In a third tab, log in to the domain's DNS service provider (e.g., Cloudflare, Amazon Route 53, etc.)
+- Examine existing records pointing to Pantheon
+  {% include("content/standard-dns-config.html") %}
+- Lower the TTL of existing DNS records to minimize impact of upcoming DNS changes
+  {% include("ttl.twig") %}
+
+## Roles & Permissions
```

Comment:
this section seems out of context?

Reply:
Maybe combine it with step 1 below?

Reply:
Changing the header here from H2 to H3, so it's still part of the "Before You Begin" section - idea being that the user intending to complete the process needs to be aware of required role first.